exchanges/graphcache/src/cacheExchange.ts

```diff
@@ -167,11 +191,11 @@ export const cacheExchange = (opts?: CacheExchangeOpts): Exchange => ({
   };
 
   // This updates the known dependencies for the passed operation
-  const updateDependencies = (op: Operation, dependencies: Set<string>) => {
-    dependencies.forEach(dep => {
+  const updateDependencies = (op: Operation, dependencies: Dependencies) => {
+    for (const dep in dependencies) {
```

Comment:
Enjoying the transformation to for loops to be fair

Reply:
same